Services/CO.CDP.OrganisationInformation/JoinRequestStatus.cs

```diff
@@ -0,0 +1,11 @@
+using System.Text.Json.Serialization;
+
+namespace CO.CDP.OrganisationInformation;
+
+[JsonConverter(typeof(JsonStringEnumConverter))]
+public enum JoinRequestStatus
```

Comment:
Suggestion, can we rename to `RequestStatus` ?

Reply:
That seems too generic to me. Could mean any kind of request then in another developer's eyes and could be confusing. HttpRequest, for example, also has a status.

Reply:
@jakzal thoughts?